### ISSUE TITLE
UseLettuceEncrypt with ConfigureKestrel not found

### ISSUE THREAD

**Issue by TomasWeg** [natemcmaster/LettuceEncrypt#127]
Here, UseLettuceEncrypt is not found. I imported LettuceEncrypt and Microsoft.Extensions/AspNetCore.Hosting but it does not found
```
.ConfigureWebHostDefaults(webBuilder =>
                {
                    webBuilder.UseStartup<Startup>();
                    webBuilder.ConfigureKestrel(options =>
                    {
                        options.ListenAnyIP(port - 1);
                        options.Listen(IPAddress.Loopback, port, options =>
                        {
                            options.UseHttps(b => b.UseLettuceEncrypt(options.ApplicationServices));
                        });
                    });
                })
```

**Comment by NoFrosty** [comment]
Got exactly the same problem : 
'HttpsConnectionAdapterOptions' does not contain a definition for 'UseLettuceEncrypt' and no accessible extension method 'UseLettuceEncrypt' accepting a first argument of type 'HttpsConnectionAdapterOptions' could be found (are you missing a using directive or an assembly reference?)

Importing this is not enough : 
using Microsoft.AspNetCore.Hosting;
using Microsoft.AspNetCore.Server.Kestrel.Https;
using Microsoft.Extensions.Hosting;

**Comment by carey-bishop** [comment]
The ```UseLettuceEncrypt()``` extension method is only available in the new pre-release builds of LettuceEncrypt (after 1.0.0).
Previous versions use ```services.AddLettuceEncrypt()``` on the ```IServiceCollection```.
See #100 and #109 for more information on the change.

**Comment by natemcmaster** [comment]
As @carey-bishop said, these API are available in the 1.1 prerelease builds, which are on NuGet.org.
